SuiteCareers/SuiteCareers/Views/Shared/_Layout.cshtml

```diff
@@ -3,12 +3,12 @@
 <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>@ViewData["Title"] - SuiteCareers</title>
+    <title>@ViewData["Title"] | SuiteCareers</title>
     <link rel="stylesheet" href="~/lib/bootstrap/dist/css/bootstrap.min.css" />
     <link rel="stylesheet" href="~/css/site.css" asp-append-version="true" />
     <link rel="stylesheet" href="~/SuiteCareers.styles.css" asp-append-version="true" />
 </head>
-<body>
+<body style="background-color: #fbfcfe">
```

Comment:
Could you put this in the main style sheet

Reply:
Sure. The styling has been moved to site.css